Remove privacy-breach links and self-host docs

Remove tracking URLs from the README and sidebar of pages
in the documentation.

--- black.orig/docs/conf.py
+++ black/docs/conf.py
@@ -19,6 +19,8 @@ from pathlib import Path
 
 from pkg_resources import get_distribution
 
+from _black_version import version as __version__
+
 CURRENT_DIR = Path(__file__).parent
 
 
@@ -43,7 +45,7 @@ author = "Łukasz Langa and contributors
 
 # Autopopulate version
 # The version, including alpha/beta/rc tags, but not commit hash and datestamps
-release = get_distribution("black").version.split("+")[0]
+release = __version__
 # The short X.Y version.
 version = release
 for sp in "abcfr":
@@ -64,13 +66,13 @@ extensions = [
     "sphinx.ext.autodoc",
     "sphinx.ext.intersphinx",
     "sphinx.ext.napoleon",
-    "myst_parser",
+#    "myst_parser",
     "sphinxcontrib.programoutput",
     "sphinx_copybutton",
 ]
 
 # If you need extensions of a certain version or higher, list them here.
-needs_extensions = {"myst_parser": "0.13.7"}
+# needs_extensions = {"myst_parser": "0.13.7"}
 
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
@@ -100,15 +102,15 @@ pygments_style = "sphinx"
 
 # We need headers to be linkable to so ask MyST-Parser to autogenerate anchor IDs for
 # headers up to and including level 3.
-myst_heading_anchors = 3
+# myst_heading_anchors = 3
 
 # Prettier support formatting some MyST syntax but not all, so let's disable the
 # unsupported yet still enabled by default ones.
-myst_disable_syntax = [
-    "myst_block_break",
-    "myst_line_comment",
-    "math_block",
-]
+# myst_disable_syntax = [
+#    "myst_block_break",
+#    "myst_line_comment",
+#    "math_block",
+#]
 
 # -- Options for HTML output -------------------------------------------------
 
@@ -129,11 +131,11 @@ html_sidebars = {
 html_theme_options = {
     "show_related": False,
     "description": "“Any color you like.”",
-    "github_button": True,
+    "github_button": False,
     "github_user": "psf",
     "github_repo": "black",
     "github_type": "star",
-    "show_powered_by": True,
+    "show_powered_by": False,
     "fixed_sidebar": True,
     "logo": "logo2.png",
 }
--- black.orig/docs/index.rst
+++ black/docs/index.rst
@@ -68,12 +68,7 @@ and `pipenv <https://docs.pipenv.org/>`_
 Show your style
 ---------------
 
-Use the badge in your project's README.md:
-
-.. code-block:: md
-
-   [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-
+Use the badge in your project's README.md.
 
 Using the badge in README.rst:
 
@@ -82,11 +77,6 @@ Using the badge in README.rst:
    .. image:: https://img.shields.io/badge/code%20style-black-000000.svg
       :target: https://github.com/psf/black
 
-Looks like this:
-
-.. image:: https://img.shields.io/badge/code%20style-black-000000.svg
-   :target: https://github.com/psf/black
-
 Contents
 --------
 
